src/apps/intel/intel_app.lua

```diff
@@ -252,13 +252,9 @@ end
 function selftest ()
    print("selftest: intel_app")
 
-   local pcideva = lib.getenv("SNABB_PCI_INTEL0") or lib.getenv("SNABB_PCI0")
-   local pcidevb = lib.getenv("SNABB_PCI_INTEL1") or lib.getenv("SNABB_PCI1")
-   if not pcideva
-      or pci.device_info(pcideva).driver ~= 'apps.intel.intel_app'
-      or not pcidevb
-      or pci.device_info(pcidevb).driver ~= 'apps.intel.intel_app'
-   then
+   local pcideva = lib.getenv("SNABB_PCI_INTEL0")
+   local pcidevb = lib.getenv("SNABB_PCI_INTEL1")
+   if not pcideva or not pcidevb then
       print("SNABB_PCI_INTEL[0|1]/SNABB_PCI[0|1] not set or not suitable.")
```

Comment:
Probably should update this printout to remove mention of SNABB_PCI0/SNABB_PCI1.  Otherwise lgtm